docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,67 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed & unsigned) broadcasted to the network. The pool is responsible for:
 
-* What is dependency?
+1) Checking if transactions are valid or invalid. For example, ensuring the account that issued the transaction has enough funds to pay the associated fees.
```

Comment:
I'd add sometihng like:
`validity` of the transaction is not hard wired to the transaction pool, but rather defined by the runtime (see `validate_transaction` function). Example checks might be:
1. Transaction Index (aka nonce) correctness
2. The account has enough funds to pay for the associated fees
3. The signature is valid

Reply:
👍